sdk/resources/Microsoft.Azure.Management.Resource/src/Generated/Models/ScriptType.cs

```diff
@@ -0,0 +1,22 @@
+// <auto-generated>
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for
+// license information.
+//
+// Code generated by Microsoft (R) AutoRest Code Generator.
+// Changes may cause incorrect behavior and will be lost if the code is
+// regenerated.
+// </auto-generated>
+
+namespace Microsoft.Azure.Management.ResourceManager.Models
+{
+
+    /// <summary>
+    /// Defines values for ScriptType.
+    /// </summary>
+    public static class ScriptType
```

Comment:
Change happened when using generate.ps. ScriptType class doesn't get generated when I run the SDK generation autorest command manually.